query-service-impl/src/main/java/org/hypertrace/core/query/service/prometheus/PromQLMetricResponse.java

```diff
@@ -0,0 +1,97 @@
+package org.hypertrace.core.query.service.prometheus;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Singular;
+
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class PromQLMetricResponse {
```

Comment:
Need to be public? guessing we're encapsulating the prometheus support in this package?

Reply:
Done